tests/datalab/test_datalab.py

```diff
@@ -1518,6 +1519,47 @@ def test_no_cluster_ids(self, data):
         assert len(lab.issue_summary["issue_type"].values) == 0
 
 
+class TestDataLabNullIssues:
+    K = 3
+    N = 100
+    num_features = 10
+
+    @pytest.fixture
+    def embeddings_with_null(self):
+        np.random.seed(SEED)
+        embeddings = np.random.rand(self.N, self.num_features)
+        # Set all feature values of some rows as null
+        embeddings[[5, 10, 22]] = np.nan
+        # Set specific feature value in some rows as null
+        embeddings[[1, 19, 25], [5, 7, 2]] = np.nan
```

Comment:
Add another `test_xyz` case that asserts that the correct rows are flagged as having issues.

Another test case would be making sure that the "most common pattern" found is correct.